### PR TITLE
[DEFECT] The wave tool creates a false-negative for the Chapman website, resulting on many ADA violations per page.

### PR DESCRIPTION
**Pull request by tatwell**
- Trello Card: https://trello.com/c/K3q9aROR/
- Issue #89

### CODE REVIEW

File: app/assets/stylesheets/layout_old/footer.scss [hunk 1]
```diff
@@ -165,97 +165,6 @@
...
-//     font-size: 11px;
-//     font-family: 'Open Sans', Sans-Serif;
-//     font-weight: 600;
-//   }
-//   @include bp(wildcard-min, 1085px) {
-//     position: relative;
-//     max-width: 990px;
-//   }
-// }
-
```
Comment (tatwell) [inline app/assets/stylesheets/layout_old/footer.scss]: Removes unexplained dead code.

**Comment by tatwell** [comment]
Deployed changes to Staging and publish three pages:

- http://dev-www.chapman.edu/index.aspx
- http://dev-www.chapman.edu/dodge/index.aspx
- http://dev-www.chapman.edu/law/index.aspx

Issue resolved in first two pages. Law is **not fixed**. I suspect this is because it is not using new templates associated with this repository's stylesheets.

Asked Product Owner (David) to advise.

**Comment by tatwell** [comment]
Production deployment will involve:

- Deploy assets.
- Update following setting in `/_files/css/style_2013.css/` (l. 1933):
    ```
    .footer-links-group > ul {
        list-style-type: none;
        padding: 0 20px 20px 20px;
        line-height: 20px;
        display: none;
        
        /* For WAVE contrast issue. */
        background-color: #eaeaea;
    }
    ```
- TBA

**Comment by mirabelrice** [comment]
👍  if we are sure that the dead code isn't there for any purpose

**Comment by mandythomas** [comment]
The ribbon height (depth?) is not the same all the way across. Above the social icons, it is slightly smaller. This is only on those pages NOT using the github master.css styles.  For example /law/index.aspx and /our-faculty/index.aspx on DEV.  There are a handful of pages not migrated to new github css styles and they are the Law homepage, and 2 of the 3 templates related to faculty directory. Individual profiles pages are ok but main our-faculty/index.aspx and a school's specific listing eg /business/faculty-research/faculty-directory.aspx are broken. Mandy will find those CSS in dev-cascade and try to make same fixes Tom made in others.

**Pull request merged**